docs/080-breaking-changes.rst

```diff
@@ -105,6 +112,28 @@ This section lists changes that might cause existing contracts to not compile an
 
 * Remove support for the ``\b``, ``\f``, and ``\v`` escape sequences in code.
   They can still be inserted via hexadecimal escapes, e.g. ``\x08``, ``\x0c``, and ``\x0b``, respectively.
+* The global variables ``tx.origin`` and ``msg.sender`` have the type ``address`` instead of
+  ``address payable``. One can convert them into ``address payable`` by using an explicit
+  conversion, i.e., ``payable(tx.origin)`` or ``payable(msg.sender)``.
+
+  This change was done since the compiler cannot determine whether or not these addresses
+  are payable or not, so it now requires an explicit conversion to make this requirement visible.
+
+* Explicit conversion into ``address`` type always returns a non-payable ``address`` type. In
+  particular, the following explicit conversions have the type ``address`` instead of ``address
+  payable``:
+
+  - ``address(u)`` where ``u`` is an arbitrary variable of type ``uint160``. One can convert ``u``
```

Comment:
Is a conversion from uint160 to address payable possible or do you have to go through address?